src/components/Navbar/style2.scss

```diff
@@ -0,0 +1,33 @@
+ @import '../../scss/variables';
+ @import '../../scss/_base';
+
+.btn {
+    padding: 8px 20px;
+    margin: 0 5px 0 5px;
+    border-radius: 1px;
+    outline: none;
+    cursor: pointer;
+
+    &:hover {
+    background-color: $primary-green-light;
+    padding: 5px 2px;
+    transition: all 0.3s ease-out;
+    color: $primary-green-dark;
+    transition: 250ms;
+    }   
+
+    &--medium {
+    background-color: $primary-green-light;
+    padding: 5px 2px;
+    border-radius: 4px;
+    font-size: 18px;
+    width: 18%;
+    color: #f4f4f5;
```

Comment:
Let's try to stick to variables that we have. If you want to make it lighter you could use opacity on the variable.